README.md

```diff
@@ -45,18 +47,18 @@ If you'd like to see some examples and learn about core use cases for Pachyderm:
 
 ## Community
 Keep up to date and get Pachyderm support via:
-- [![Twitter](https://img.shields.io/twitter/url/https/twitter.com/pachydermio.svg?style=social&label=Follow%20%40pachydermio)](https://twitter.com/pachyderminc) Follow us on Twitter.
+- [![Twitter](https://img.shields.io/twitter/follow/pachyderminc?style=social)](https://twitter.com/pachyderminc) Follow us on Twitter.
 - [![Slack Status](http://slack.pachyderm.io/badge.svg)](http://slack.pachyderm.io) Join our community [Slack Channel](http://slack.pachyderm.io) to get help from the Pachyderm team and other users.
 
-### Contributing
+## Contributing
 
 To get started, sign the [Contributor License Agreement](https://pachyderm.wufoo.com/forms/pachyderm-contributor-license-agreement).
```

Comment:
Does this wufoo CLA point to the right place? I know we have it living somewhere else for the CLA bot, but idk how these integrate or not? If they dont, we need to update the CLA in both places?